app/controllers/causes_controller.rb

```diff
@@ -5,7 +5,21 @@ class CausesController < ApplicationController
   def update
     @cause.update(cause_params)
     @fmea = @cause.failure_mode.function.fmea
-    redirect_to edit_fmea_path(@fmea, anchor: card_id(@cause))
+    # redirect to the function where the cause was added
+    redirect_to edit_fmea_path(@fmea, anchor: card_id(@cause.failure_mode.function))
```

Comment:
small improvement to provide a bit of vertical space when we are redirected back to the edit fmea page